src/docs/product/accounts/sso/okta-sso/okta-scim.mdx

```diff
@@ -65,6 +65,16 @@ As a result of these changes, users who are assigned will be sent an invitation
 
 You can use "Push Groups" to sync and assign groups in Okta; they will be reflected in Sentry teams.
 
+<Alert level="note">
+
+If you use Okta to assign organization membership, you’ll be unable to make membership changes through Sentry and will need to continue using Okta.
+
+</Alert>
+
+![Okta Provisioned User Unable to Leave](./okta-provisioned-unable-to-leave.png)
```

Comment:
Remove vs Leave is the same thing, just different copy. You can get away with just showing "Leave".